jc/parsers/proc_net_tcp.py

```diff
@@ -0,0 +1,165 @@
+"""jc - JSON Convert `/proc/net/tcp` file parser
+
+Usage (cli):
+
+    $ cat /proc/net/tcp | jc --proc
+
+or
+
+    $ jc /proc/net/tcp
+
+or
+
+    $ cat /proc/net/tcp | jc --proc-net-tcp
+
+Usage (module):
+
+    import jc
+    result = jc.parse('proc', proc_net_tcp_file)
+
+or
+
+    import jc
+    result = jc.parse('proc_net_tcp', proc_net_tcp_file)
+
+Schema:
+
+    [
+      {
+        "sl":                           string,
+        "local_address":                string,
+        "rem_address":                  string,
+        "st":                           string,
+        "tx_queue":                     string,
+        "rx_queue":                     string,
+        "tr":                           string,
+        "tm->when":                     string,
+        "retrnsmt":                     string,
+        "uid":                          string,
+        "timeout":                      string,
+        "inode":                        string,
+        "sock_ref_count":               string,
+        "sock_mem_loc":                 string,
+        "retransmit_timeout":           string,
+        "soft_clock_tick":              string,
+        "(ack.quick<<1)|ack.pingpong":  string,
```

Comment:
Could we rename the `tm->when` and `(ack.quick<<1)|ack.pingpong` fields. I try to keep field-names to only alphanumerics and underscores.

Reply:
Also, should any of these values be converted to ints or floats?